src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/ContentPicker/ContentPickerEmpty/ContentTable/ContentTable.container.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import DSContentTable from '../../../../../../../../DesignSystem/ContentTable/ContentTable';
+import {useQuery} from 'react-apollo-hooks';
+import {translate} from 'react-i18next';
+import {ProgressOverlay} from '@jahia/react-material';
+import * as PropTypes from 'prop-types';
```

Comment:
Avoid import * syntax.

```suggestion
import PropTypes from 'prop-types';
```

is better